dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/container/DhisPostgreSQLContainer.java

```diff
@@ -57,6 +57,7 @@ protected void configure()
         addEnv( "POSTGRES_USER", getUsername() );
         addEnv( "POSTGRES_PASSWORD", getPassword() );
         setCommand( getPostgresCommandWithCustomConfigs() );
+        withInitScript( "db/extensions.sql" );
```

Comment:
Is this only needed by the one test that was added in this PR? Do you know how long it takes to create the extensions? Just asking so we do not incur to much overhead in all our integration tests for something we only need once.

Reply:
It is a straightforward extension creation script. Should be comparable to a create statement I expect.